{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "e8ca230e_4df8646d",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 3
      },
      "lineNbr": 0,
      "author": {
        "id": 1000011
      },
      "writtenOn": "2025-03-06T19:59:11Z",
      "side": 1,
      "message": "Thanks for working on this!",
      "revId": "bca6bf914a97853b125d009de317f33e8a20f2be",
      "serverId": "40b9299a-d8a8-485d-9b01-e6d3f45eefb5"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "84447411_ec9d0b07",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 3
      },
      "lineNbr": 0,
      "author": {
        "id": 1000011
      },
      "writtenOn": "2025-03-06T20:00:33Z",
      "side": 1,
      "message": "Jim: Do you suppose any of the other open NFS4 tickets to really have been duplicates of #16924, or had the same underlying cause, and we can close them too? Or at least ask users to retest?",
      "parentUuid": "e8ca230e_4df8646d",
      "revId": "bca6bf914a97853b125d009de317f33e8a20f2be",
      "serverId": "40b9299a-d8a8-485d-9b01-e6d3f45eefb5"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "0365bded_393cb9dc",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 3
      },
      "lineNbr": 0,
      "author": {
        "id": 1000011
      },
      "writtenOn": "2025-03-06T20:03:29Z",
      "side": 1,
      "message": "To me, #13843 at least looks like a dupe.",
      "parentUuid": "84447411_ec9d0b07",
      "revId": "bca6bf914a97853b125d009de317f33e8a20f2be",
      "serverId": "40b9299a-d8a8-485d-9b01-e6d3f45eefb5"
    },
    {
      "unresolved": false,
      "key": {
        "uuid": "547c6ed5_a6ccc0c8",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 3
      },
      "lineNbr": 0,
      "author": {
        "id": 1000366
      },
      "writtenOn": "2025-03-10T23:17:36Z",
      "side": 1,
      "message": "I\u0027m not sure that any of the other tickets are fixed by this patch.  The problems caused by #16924 seemed to occur during file operations such as open, stat, and fsync.  For #13843, somehow the file was successfully opened already before the KDL happened.  If the node was stale, I don\u0027t know how the driver was able to open it in the first place.",
      "parentUuid": "0365bded_393cb9dc",
      "revId": "bca6bf914a97853b125d009de317f33e8a20f2be",
      "serverId": "40b9299a-d8a8-485d-9b01-e6d3f45eefb5"
    }
  ],
  "submitRequirementResults": [
    {
      "submitRequirement": {
        "name": "Build-Check",
        "description": {
          "value": "Builds OK on all architectures"
        },
        "applicabilityExpression": {
          "value": {
            "expressionString": "-branch:refs/meta/config"
          }
        },
        "submittabilityExpression": {
          "expressionString": "-label:Verified\u003d-1"
        },
        "overrideExpression": {},
        "allowOverrideInChildProjects": false
      },
      "applicabilityExpressionResult": {
        "value": {"expression":{"expressionString":"-branch:refs/meta/config"},"status":"PASS","errorMessage":{"value":null},"passingAtoms":[],"failingAtoms":["branch:refs/meta/config"]}
      },
      "submittabilityExpressionResult": {
        "value": {"expression":{"expressionString":"-label:Verified=-1"},"status":"PASS","errorMessage":{"value":null},"passingAtoms":[],"failingAtoms":["label:Verified=-1"]}
      },
      "overrideExpressionResult": {},
      "patchSetCommitId": "bca6bf914a97853b125d009de317f33e8a20f2be",
      "legacy": {
        "value": false
      },
      "forced": {},
      "hidden": {}
    }
  ]
}